src/OrchardCore.Modules/OrchardCore.Recipes/Services/RecipeDeploymentTargetHandler.cs

```diff
@@ -32,6 +42,9 @@ public async Task ImportFromFileAsync(IFileProvider fileProvider)
                 RecipeFileInfo = fileProvider.GetFileInfo("Recipe.json")
             };
 
+            var environment = new Dictionary<string, object>();
+            await _environmentProviders.InvokeAsync((provider, env) => provider.SetEnvironmentAsync(env), environment, _logger);
+
             await _recipeExecutor.ExecuteAsync(executionId, recipeDescriptor, new Dictionary<string, object>(), CancellationToken.None);
```

Comment:
You need to pass the `environment` variable

Reply:
I did the change

Reply:
ty :)